.github/workflows/release.yml

```diff
@@ -0,0 +1,74 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        required: true
+        description: Version (Semver without leading v)
+  push:
+    paths-ignore:
+      # - '**/*.md'
+      - 'src/docs/**/*'
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - 'v[0-9]+.[0-9]+.[0-9]+-alpha'
+      - 'v[0-9]+.[0-9]+.[0-9]+-beta'
+      - 'v[0-9]+.[0-9]+.[0-9]+-rc'
+
+jobs:
+  release-version:
+    env:
+      PACKAGE_ID: "KungFunc.CLI.Demo"
```

Comment:
Is it intentional that the name is `KungFunc`?

Reply:
Nope...this has to be dynamic or removed...Let me check

Reply:
@dspetrov  - good catch. This is left-over from the Demo repo testing. 

Also forgot to finish the renaming of the Endatix.SqlServer to Endatix.Persistence.SqlServer as agreed previously. Now it's done

Another issue I have found was that the Endatix.Api.Host was not added to the solution, which I have fixed and it's OK now

Reply:
Great! All seems fine now.